pkg/renderer/templates/public_values.yaml.tpl

```diff
@@ -166,6 +166,7 @@ scanner:
   {{- end }}
 
 scannerV4:
+  disable: false
```

Comment:
Won't this enable scanner V4 if you run `helm [...] -f values-public.yaml`?

Reply:
Thank you, good catch. Just pushed a commit to fix this, let's see what CI thinks about it.